LiskHQ/lisk-sdk
# Update synchronizer to typescript - Closes #5192

### What was the problem?

This PR resolves #5192 

### How was it solved?

- Update synchronizer to typescript

### How was it tested?

- Check all eslint issue is resolved for synchronizer
- Check all tests are passing
